workspace_host/interface.js

```diff
@@ -367,11 +367,11 @@ async.series(
       });
     },
   ],
-  function (err, data) {
+  async function (err, data) {
     if (err) {
       Sentry.captureException(err);
       logger.error('Error initializing workspace host:', err, data);
-      markSelfUnhealthy(err);
+      await markSelfUnhealthy(err);
```

Comment:
This function will never throw, so I'm inclined to just disable the rule here. If you do indeed want the await, I don't think it's valid to pass an `async` function as the last argument to `async.series`, so you could write it like this instead, as `async.series` will return a Promise if no callback is provided:

```js
async.series([
  // ...
  async () => logger.info('Workspace host ready'),
]).catch(async (err) => {
  Sentry.captureException(err);
  logger.error('Error initializing workspace host:', err, data);
  await markSelfUnhealthy(err);
});
```